plugins/quickbooks-online/src/manifest.json

```diff
@@ -37,6 +37,36 @@
       "description": "Income Account number from QuickBooks",
       "required": 1
     },
+    {
+      "key": "qbServicePlanIncomeAccountNumber",
+      "label": "Service Plan Income account number",
+      "description": "Income Account number for service plans from QuickBooks",
+      "required": 1
```

Comment:
Hello @mrinc, sorry for the review delay.
Please do not make the new configuration items required, it's not needed. When not filled, you can just skip the processing of these.